.idea/codeStyleSettings.xml

```diff
@@ -133,7 +133,6 @@
           <option name="ALIGN_MULTILINE_THROWS_LIST" value="true" />
           <option name="ALIGN_MULTILINE_EXTENDS_LIST" value="true" />
           <option name="ALIGN_MULTILINE_PARENTHESIZED_EXPRESSION" value="true" />
```

Comment:
Please do not commit codeStyleSettings changes. We use different IDEA versions and it's supposed that current xmls should be applicable to IDEA built from master.